src/adapter.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const { Adapter } = require('interface-transport')
```

Comment:
Cool!

Reply:
We decided after merging TCP to fall the adapter as we will release `js-libp2p` with the async migration once everything else is ready

Reply:
Less Code™